docs/osquery/osquery.asciidoc

```diff
@@ -54,7 +54,13 @@ image::images/enter-query.png[Select saved query dropdown name showing query nam
 +
 TIP: To save a single query for future use, click *Save for later* and define the ID, description, and other <<osquery-manage-query,details>>.
 
-. Review the results. Next, navigate to *Discover* to dive deeper into the response or to *Lens* to create visualizations.
+. Review the results and do any of the following:
+
+** Click *View in Discover* (image:images/discover-button-osquery.png[Click the View in Discover button,20,20]) to explore the results in Discover.
+** Click *View in Lens* (image:images/lens-button-osquery.png[Click the View in Lens button,20,20]) to navigate to Lens, where you can use the drag-and-drop *Lens* editor to create visualizations.
+** Click *Add to Case* (image:images/case-button-osquery.png[Click Add to Case button,20,20]) to add the query results to a new or existing case. 
+** Click the view details button (image:images/view-osquery-details.png[Click View details button,20,20]) to examine the query ID and statement.  
```

Comment:
Alt text typically doesn't include a command to the reader, because the body text already tells them what to do, click, etc. Just a description of the image is enough.

Also, revising to "[icon](https://docs.elastic.dev/content-design/tech-docs/writing-about-ui#icon-or-button)" since the UI elements don't have text labels:

```suggestion
** Click *View in Discover* (image:images/discover-button-osquery.png[View in Discover icon,20,20]) to explore the results in Discover.
** Click *View in Lens* (image:images/lens-button-osquery.png[View in Lens icon,20,20]) to navigate to Lens, where you can use the drag-and-drop *Lens* editor to create visualizations.
** Click *Add to Case* (image:images/case-button-osquery.png[Add to Case icon,20,20]) to add the query results to a new or existing case. 
** Click the view details icon (image:images/view-osquery-details.png[View details icon,20,20]) to examine the query ID and statement.  
```